lib/iris/tests/test_netcdf.py

```diff
@@ -240,6 +250,52 @@ def test_cell_methods(self):
 
         self.assertCML(cubes, ("netcdf", "netcdf_cell_methods.cml"))
 
+    def test_ancillary_variables(self):
```

Comment:
It might also be worthwhile also testing for cell measures. As far as I can tell, they aren't currently being tested for.

Reply:
Ok..